blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,20 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
-		'category'    => $attributes['categories'],
-	) );
+	/** This filter is documented in wp-includes/widgets/class-wp-widget-recent-posts.php of WordPress core codebase */
```

Comment:
This should be on the line before the `apply_filters()` call. See https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
Apologies. I was assuming people would connect the dots down a single line.